egs2/librispeech/wavlm1/local/data_pretraining.sh

```
@@ -0,0 +1,160 @@
+train_set=pretrain_train
+valid_set=pretrain_dev
+dev_per_dset=150          # utterances held out per corpus/language
+audio_ext=flac
+dumpdir=dump/raw/org
+scandir=data/.audio_scan     # cached filesystem listing, one file per corpus/lang
+fs=16000
+nj=8
+stage=0
+stop_stage=3
+
+. utils/parse_options.sh
```

Shellcheck (0.11.0)

[info] Not following: utils/parse_options.sh was not specified as input (see shellcheck -x).

(SC1091)